qiskit/circuit/library/arithmetic/adder/reversibleadder.py

```diff
@@ -0,0 +1,82 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals
+
+"""Module-level docstring describing what the file content is."""
+
+
+from qiskit import QuantumRegister, ClassicalRegister, QuantumCircuit
+from qiskit import BasicAer
+from qiskit import execute
```

Comment:
You only need to import `QuantumCircuit` and `QuantumRegister`, since the other classes are all not needed, right?

Reply:
Sure, will remove the other classes